fibbingnode/res/default.cfg

```diff
@@ -5,12 +5,25 @@ cost=1
 # ospf dead interval timer
 dead_interval=40
 # ospf hello interval timer
-hello_interval=10
+hello_interval=20
```

Comment:
This will slow down _a lot_ the convergence time at bootstrap, which is not something we want for quick experiments